src/cli/generate.ts

```diff
@@ -3,18 +3,28 @@ import * as yargs from 'yargs';
 
 import { generate } from '../Command/Generate';
 import FileManager from '../FileManager';
+import * as fs from 'fs';
+import { exit } from 'process';
+
+let pwd = process.env.PWD+'/';
+
+if(pwd === undefined){
+  console.warn('Print working directory not defined. Ensure you pass a value for sourceDir (s).')
+  pwd = '/';
+}
+
 
 const argv = yargs.options({
   sourceDir: {
-    type: 'array',
+    type: 'string', //Upstream 1.4 supports arrays as input. Our fork will not because, we parse the directories differently.
     alias: 's',
-    demandOption: true,
-    describe: 'The directory or directories location(s) which contains your apex .cls classes.',
+    default: pwd,
+    describe: 'The root directory of your salesforce project. The sfdx-project.json determines the class folder.',
```

Comment:
I have a few thoughts around the SFDX change:

- Believe or not I still work with some legacy projects that haven't been migrated to SFDX (so they don't have an sfdx-project.json). Unfortunately these can't be migrated just yet (they can but it is easier to not touch them for different reasons: legacy code, 1st gen managed packages, etc.). I imagine that's the case for others out there so I'd still want this tool to be able to support those kind of projects
- The way the current tool works it traverses the directory structure recursively anyways, so even if they are SFDX projects they should still be supported. What would be the advantage of introducing this change?